styleguide/source/_patterns/01-molecules/06-components/20-card-image-landscape/_card-image-landscape.scss

```diff
@@ -1,98 +1,53 @@
-.layout_item {
-  position: relative;
-}
-
-.card {
-  display: flex;
-  flex-direction: column;
-  background-color: $black-7;
-  color: $black;
-  //margin-bottom: 20px;
-  text-align: center;
-  height: 100%;
-}
-
 .card-image-landscape {
-  background-size: cover;
-  background-position: center;
-  //min-height: calc((100vw - 30px) / 3 * 2);
+  display: block;
+  position: relative;
+  width: 100%;
 
-  @include breakpoint($bp-small) {
-    //min-height: calc( (100vw - 40px - 80px) / 3 * 2 / 3 );
+  @include breakpoint($bp-med) {
+    max-width: 600px; //don't grow larger than the image width
   }
 
-  @include breakpoint($bp-med) {
-    //min-height: calc( (100vw - 40px - 80px) / 3 * 2 / 4 );
+  &.card {
+    color: $black;
+    text-align: center;
+    height: 100%;
+
+    &:hover {
+      opacity: 0.8;
+    }
   }
 }
 
-.card.card-image-landscape:hover {
-  opacity: 0.8;
+.card-image-landscape_image {
+  width: 100%;
 }
 
 // @todo - this is repeating the "overlay" on other CTA elements.
 // @todo - it should be an atom if this is the case.
 // @todo - also probably shouldn't be specific to landscape cards.
-.card-image-landscape .card-image-landscape_overlay {
-  margin-top: auto;
+.card-image-landscape_overlay {
   background-color: rgba(255, 255, 255, 0.45);
-  padding: 24px 0 12px;
+  bottom: 0;
+  padding: 24px 0;
+  position: absolute;
+  width: 100%;
 
   * {
     margin: 0 24px;
   }
 }
 
-.card-image-landscape .card-image-landscape_title {
+.card-image-landscape_title {
   @extend %type-larger;
 }
 
-.card-image-landscape .card-image-landscape_description {
-  @include type($font-sans-serif, 18px, $font-weight-medium, 1.3888888889);
-
-  @include breakpoint($bp-med) {
-    @include type($font-sans-serif, 20px, $font-weight-medium, 1.3333333333);
-  }
-}
-
-// Styling for image landscape cards when they're included in rows.
+.card-image-landscape_description {
 
-.layout_three_up-row > *,
```

Comment:
Since we have separate row layouts for 3, 4, and 5 up and these are landscape and don't have the same height issues as some other cards, I don't think we actually need these styles which is why I removed them.